docs/source/algorithms/MultiplyRange-v1.rst

```diff
@@ -18,7 +18,7 @@ Usage
 
   # Create a workspace with 100 bins
   ws = CreateSampleWorkspace()
-  # Multuply the values in bins 25 to 75 by 2
+  # Multuply the values in bins 25 to 75 by 3
```

Comment:
```suggestion
  # Multiply the values in bins 25 to 75 by 3
```

Reply:
Sorry for force pushing, I've added this to the commit